Live/Classes/MainScene.cpp

```diff
@@ -1,5 +1,6 @@
 #include "MainScene.h"
 #include "SimpleAudioEngine.h"
+#include "limits.h"
```

Comment:
is this a cocos library file? or from the standard library? change to `<limits.h>` if its from standard library.

Reply:
removed